### ISSUE TITLE
[FEATURE REQUEST]  Keyboard Shorcuts / commands support

### ISSUE THREAD

**Issue by Marin37** [mrdokenny/Cookie-AutoDelete#1221]
### Acknowledgements

- [X] I acknowledge that I have read the above items

### Is your feature request related to a problem? Please describe.

Autodelete already has a variety of settings to do things automatically, but as a user I'd like to have the option to add custom shortcuts to perform some of the extension's **manual actions**. Even though most actions are 2 or 3 clicks away, some would find this useful for productivity or simply just convenience and customization (for me, I just like avoiding the mouse whenever possible).

### Describe the solution that you'd like

A great example of this are the assignable shortcuts from the ublock origin extension _(in fact, these aren't on the extension page/config but rather as assignable shortcuts from firefox's addons manager)_. 
![image](https://user-images.githubusercontent.com/57327931/132430378-6950aa90-c198-48e4-9354-18c393dbb2b7.png)

### Describe alternatives you've considered

Another alternative would be to have some terminal/shell/api (I'm not sure if it is possible) that could be able to process some commands as the extension's features on input. This _alone_ may not necessarily be the most convenient for what I'm pointing out, but in conjunction with other _(third party)_ extensions that can be configured to have some X shortcut that sends Y command to other extensions it would _(with a third party addon)_ solve this issue, even gesturefy users could use "Send message to other addon" with their mouse to perform this.

### Additional Context

My specific use case I'm pointing out is from one of my school's login page, which has its own administration/matriculation/register page but for the actual class, materials and excersises it redirects to Moodle. We do not have information about our Moodle account, but rather every time we go in we have to first go to the school's own platform and only then we click a button and redirects to the Moodle's class.
Once I enter to the school's system, I can fully use Moodle, close it and reopen it without any problem for some period of time. After that, Moodle simply closes my session and gives me nothing but a login-required-like page (but we do not know our user info). The school's page does not, however, close my session at all, and if I click the redirect button again it ends the same as just described. So, a bunch of times a day i manually use `Clean All for this domain` on the first page to login again (and being able to access both platforms altogether) - _Here's where a shortcut setting/feature may be pretty useful for me_

I searched some of the feature requests, and there's a similar one open at https://github.com/Cookie-AutoDelete/Cookie-AutoDelete/issues/46, yet I'm adding this one as I'm pointing alternative/use case/extra info which I don't think it would be appropiate on it (please correct me if I'm wrong!)
So, TLDR; I'm just asking for a bunch of **shortcuts** for the **manual** features within the extension.
Anyways, thanks for such an already usefull and handy program!

**Comment by kennethtran93** [comment]
This can be merged into #46.
